_maps/map_files/generic/CentCom_skyrat_z2.dmm

```diff
@@ -1509,11 +1509,11 @@
 /turf/open/floor/iron/dark,
 /area/cruiser_dock)
 "aqu" = (
-/obj/item/toy/plush/arcplush{
-	desc = "A small plushie that resembles a, quote, nerd - according to a crudely scribbled tag stapled onto the plush.";
-	name = "nerd plushie"
-	},
 /obj/structure/fluff/beach_umbrella/engine,
+/obj/item/paper{
+	info = "The Nerd plushie has gone on an extended vacation. You may keep her umbrella, though, she may be back. - A.D.X.";
+	name = "Nerd Plushie Note"
+	},
```

Comment:
I still don't see the point in leaving a note for this. If people want to know where it went, they can look at the repo/code, or just ask.
```suggestion
```

Reply:
It's just a note for the plushie, pff, seeing as it's been there since the start of that ghost cafe map